[Nexus] LTS update (#6803)
captn3m0/endoflife.date: products/nexus.md

with updates and bug fixes, followed by 6 months of extended maintenance, with only critical and
security fixes. After this, the release is considered sunset.

> In release 3.71.0, Sonatype Nexus Repository began using H2 as its default embedded database. If you are not able to migrate to H2 or PostgreSQL, you will need to remain on our [3.70.x](https://help.sonatype.com/en/orientdb-downloads.html) release line, which will receive critical bug fixes.

Reviewer: ```suggestion
In release 3.71.0, Sonatype Nexus Repository began using H2 as its default embedded database. Users not able to migrate to H2 or PostgreSQL need to remain on the [3.70.x](https://help.sonatype.com/en/orientdb-downloads.html) release line, which will receive only critical bug fixes.
```
